### ISSUE TITLE
Documentation issue for duplicates()

### ISSUE THREAD

**Issue by fgabolde** [perl5-utils/List-MoreUtils#36]
The POD for `duplicates` asserts:

> In scalar context, returns the number of elements occurring only once in LIST.

Should be "more than once".

**Comment by rehsack** [comment]
Seems to but a duplicate of #33 :)

**Comment by fgabolde** [comment]
Ah my bad, I had checked only the open issues before creating this one!
